### PR TITLE
feature: add dataset button to analytics

### PR DESCRIPTION
**Pull request by vid277**
image

### CODE REVIEW

**Comment by drew-harris** [comment]
I would move the dashboard icon to the bottom of that box to show that it's on a separate site. It looks like just another page on the analytics when its not.  Could also but a link to the api docs there too.

**Comment by drew-harris** [comment]
Like here
image

**Comment by vid277** [comment]
image

File: .env.example [hunk 1]
```diff
@@ -24,6 +24,11 @@ VITE_DASHBOARD_URL="http://localhost:5173"
 ##### SEARCH #####
 VITE_SENTRY_SEARCH_DSN=""
 
+##### ANALYTICS #####
+VITE_API_HOST=http://localhost:8090/api
+VITE_SEARCH_UI_URL="http://localhost:5174"
+VITE_DASHBOARD_UI_URL="http://localhost:5173"
+
```
Comment (cdxker) [inline .env.example:31]:
```suggestion
```
These shouldn't be needed, as they are already defined in other locations.

**Review by cdxker — CHANGES_REQUESTED** [review]
One comment request and can you change the env name to `VITE_DASHBOARD_URL`, just to be consistent with how its named in the search ui and dashboard UI. Other than that it looks great!

**Review by densumesh — APPROVED** [review]

**Pull request merged**